.travis.yml

```diff
@@ -122,6 +122,9 @@ before_script:
   - tar xzf tar-output/ledgersmb-travis.tar.gz
   - cd ledgersmb
   - cp conf/ledgersmb.conf.default ledgersmb.conf
+  - if [ "${DOJO_BUILT}" != "" ]; then
+      sed -r -i -e "s/^( |\t|#)*dojo_built( |\t)*=.*/dojo_built = ${DOJO_BUILT}/" ledgersmb.conf ;
+    fi
```

Comment:
I'll solve the maintenance issue with a test to confirm all 3 are in sync (default, unbuilt, and travis)
at the same time, I'll make sure they are all in synv